server/bed.js

```diff
@@ -14,9 +14,12 @@ function bed() {}
 
 /**
  * @typedef {{
- *   chrStart: number,
- *   chrEnd: number,
- *   label: string
+ *   aggregated: boolean,
```

Comment:
The previous struct is reasonably named Motif. But the current version is more like 'bed.query.MotifsResult'.